setup_creators/solutions/scripts/plot-positions.py

```
@@ -0,0 +1,28 @@
+import matplotlib
+
+matplotlib.use("Agg")  # to suppress interactive plotting
+import matplotlib.pyplot as plt
+import numpy as np
+from pysam import VariantFile
+import seaborn as sns
+
+window_size = 500
+
+pos = [record.pos for record in VariantFile(snakemake.input[0])]
```

Ruff

Undefined name `snakemake`

(F821)